backend/instancemgmt/instance_manager.go

```diff
@@ -17,6 +18,7 @@ var (
 		Name:      "active_instances",
 		Help:      "The number of active plugin instances",
 	})
+	disposeTTL = 5 * time.Second
```

Comment:
Lets discuss what this should be

Reply:
if this TTL is to avoid multiple instances calling  `Get` at the same time I think 5s is a good value

Reply:
Cool lets start with this and evaluate if it works or not